assets/js/googlesitekit/datastore/user/authentication.js

```diff
@@ -239,6 +239,22 @@ export const selectors = {
 		const { unsatisfiedScopes } = select( STORE_NAME ).getAuthentication() || {};
 		return unsatisfiedScopes;
 	} ),
+
+	/**
+	 * Checks reauthentication status for this user.
+	 *
+	 * Returns true if any required scopes are not satisfied or undefined
+	 * if reauthentication info is not available/loaded.
+	 *
+	 * @since 1.9.0
```

Comment:
Should be `@since n.e.x.t`, exact version numbers will always be added as part of the release preparation.

Reply:
Done